### DISCUSSION TITLE
graphql/config/docs/user/schema

### DISCUSSION THREAD

**Discussion by giscus** [kamilkisiela/graphql-config#1408] [Docs Discussions]
# graphql/config/docs/user/schema

GraphQL Config Documentation

https://the-guild.dev/graphql/config/docs/user/schema

**Comment by wolffparkinson** [comment]
How to poll url schema for changes ?

**Comment by koyta** [comment]
How tf make IDE's plugins to read .env.local in order to get value for, e.g., authorization token? In Intellij it requires custom input, in VS Code it's not even supported...

**Reply by MangoMarcus** [reply]
@kotya Don't know if it helps but I _eventually_ got it to work in vscode by manually initialising the env with dotenv

```ts
// graphql.config.ts
import type { IGraphQLConfig } from 'graphql-config';
import path from 'path';

// https://github.com/graphql/graphiql/issues/2359#issuecomment-1102937944
import { config as initEnv } from 'dotenv';
// Can't use path.resolve(process.cwd(), '.env.local') as vscode doesn't seem to run this from the project root
initEnv({ path: path.join(__dirname, '.env.local') });
// Allow mkcert certs for dev (don't do this on prod, might need to see if there's a better workaround)
process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';

const GRAPHQL_URL = process.env.GRAPHQL_URL as string;
const config: IGraphQLConfig = {
  schema: GRAPHQL_URL,
};

export default config;
```

**Comment by Manuel2u** [comment]
After specification of your config file , what next

**Comment by addiebarron** [comment]
I'd expect code files to be supported as schema files as well, like [graphql-codegen](https://the-guild.dev/graphql/codegen/docs/config-reference/schema-field#code-files) does. Made a ticket for this [over here](https://github.com/kamilkisiela/graphql-config/issues/1425).
